libs/auth/README.md

```diff
@@ -0,0 +1,24 @@
+# Auth
```

Comment:
This readme should be directed at the dev who uses daffodil to build an app, rather than at a dev who is developing daffodil. There's a "daffodil dev" document in the root of the repo under "docs/", and there's contributor guidelines in the root of the repo as well. Check out https://github.com/graycoreio/daffodil/pull/566/files#diff-95bd3aaa2e3fc23ada2bca09245b811c for an example.